src/Sarif.Converters/PluginConverterFactory.cs

```diff
@@ -32,7 +39,7 @@ public override ToolFileConverterBase CreateConverterCore(string toolFormat)
                 throw new ArgumentException(message, nameof(this.pluginAssemblyPath));
             }
 
-            Assembly pluginAssembly = Assembly.LoadFile(this.pluginAssemblyPath);
+            Assembly pluginAssembly = this.assemblyLoadFileDelegate(this.pluginAssemblyPath);
```

Comment:
>this. [](start = 68, length = 5)

Should the 'this' keyword be present outside of the constructor for this variable?

Reply:
In brief, yes. Our convention is to use `this.` for instance fields (check with Michael to confirm). The SARIF SDK code base does not follow this convention consistently.